calculator4.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
```

Comment:
Add error handling for non-numeric inputs. You can use the isinstance() function to check if the inputs are numbers before performing the operations. [important]